### PR TITLE
Several fixes for the Python 3 version

### PR DESCRIPTION
**Pull request by reutenauer**
See discussion from [this comment](/khaledhosny/dviasm/issues/7#issuecomment-553879420) onwards for details.

### CODE REVIEW

**Comment by reutenauer** [comment]
Sorry for the conflicts, it was an ill-advised rebase in my repository.  I can send a new pull request if you want.

File: dviasm.py [hunk 1]
```diff
@@ -679,9 +688,8 @@ def WriteFontDefinitions(self, fp):
         s.append(PutSignedQuad(self.font_def[e]['scaled_size']))
         s.append(Put2Bytes(flags))
         s.append(PutByte(len(self.font_def[e]['name'])))
-        s.append(self.font_def[e]['name'])
+        s.append(self.font_def[e]['name'].encode('utf8'))
         s.append(PutSignedQuad(self.font_def[e]['index']))
-        print >> sys.stderr, self.font_def[e]['name'], self.font_def[e]['index']
```
Comment (reutenauer) [inline dviasm.py]: This line was originally introduced in [that commit](/khaledhosny/dviasm/commit/c3487d0) over five years ago, it generates a lot of output in some cases, which I don’t think is useful anymore.

Comment (khaledhosny) [reply dviasm.py]: Right, that shouldn’t have been committed in the first place.

**Review by khaledhosny — APPROVED** [review]
> Sorry for the conflicts, it was an ill-advised rebase in my repository. I can send a new pull request if you want.

`git rebase -i` is your friend. I rebased and pushed to your repo so I can merge this from GitHub UI.

**Comment by khaledhosny** [comment]
(you might want to `git fetch` from this repo and `git reset --hard origin master` to avoid conflicts in the future).

**Comment by reutenauer** [comment]
Thanks

**Comment by reutenauer** [comment]
I wanted to do `rebase -i`, actually, but must have botched it somehow.

**Pull request merged**